vignettes/ectd/adam-adadas.R

```diff
@@ -0,0 +1,133 @@
+## ---- message = FALSE-----------------------------------------------------------------------------------------------
+# Initiate start-up file
+source(file.path(rprojroot::find_root("DESCRIPTION"), "inst/startup.R"))
+
+## -------------------------------------------------------------------------------------------------------------------
+# Working directory requires write permission
```

Comment:
We should agree on a standard workflow at the top of the program. Typically you dont need to initialize the startup.R and do a check for the writing permission.

I only have two lines:

# Set up ------------------------------------------------------------------

fcts <- c("eff_models.R", "fmt.R", "helpers.R", "Tplyr_helpers.R")
invisible(sapply(fcts, FUN = function(x) source(file.path("R/", x), )))

Reply:
Agree, no need for sourcing the startup script and checking write permissions. I think we don't even need the two lines you suggested @SHAESEN2. Since the project is a R package and the functions are in the `R/` folder we just have to `library()` the pakage to make the function available.

Reply:
> Thanks @SHAESEN2 @thomas-neitmann for the review! I've addressed the above comments, also added program headers as @SHAESEN2 did in `submission/programs/adtte.R`.
> 
> Please suggest on folder structures for `.R` and output xpt dataset. I put `adadas.R` and `qc_adadas.R `under vignettes/ectd/ for now as discussed in the last meeting.

@kaz462 I proposed a folder structure for the dataset and program in #21 #27 waiting for team approval. Not sure where the qc program should live. Perhaps a qc folder in root?